docs/io/fits/appendix/faq.rst

```diff
@@ -325,7 +325,11 @@ like so:
         # (this example is assuming a 64-bit float)
         # The -1 is to account for the final byte that we are about to
         # write:
-        fobj.seek(len(header.tostring()) + (40000 * 40000 * 8) - 1)
+        file_length = len(header.tostring()) + (40000 * 40000 * 8) - 1
```

Comment:
This is not quite right: you need to do the `- 1` at the very end, otherwise the file is going to be 1 byte too long!
You can also avoid the `if` statement (though perhaps that is less clear?):
```
file_length = len(header.tostring()) + (40000 * 40000 * 8)
# FITS files must be a multiple of 2880 bytes long; the -1
# is to account for the final byte that we are about to write.
file_length = ((file_length + 2880 - 1) // 2880) * 2880 - 1
```

Reply:
This is much better. Thank you for the review and suggestions.